node/primitives/src/lib.rs

```diff
@@ -405,3 +412,227 @@ impl SignedDisputeStatement {
 		self.session_index
 	}
 }
+
+/// A `DisputeMessage` where signatures of statements have not yet been checked.
```

Comment:
This is a network message, right? Perhaps the docs or type should clarify that.

Reply:
Only indirectly, but yeah I will add a comment about its use.